ambari-agent/src/main/python/ambari_agent/alerts/web_alert.py

```diff
@@ -20,9 +20,9 @@
 
 import logging
 import time
-import urllib2
+import urllib.request, urllib.error, urllib.parse
 
-from urllib2 import HTTPError
+from urllib.error import HTTPError
 
 from tempfile import gettempdir
 from alerts.base_alert import BaseAlert
```

Comment:
`alerts.base_alert` => `ambari_agent.alerts.base_alert`
Maybe it should be also changed?

Reply:
both these two way are work.